plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/resource/LibvirtDomainXMLParserTest.java

```diff
@@ -259,4 +259,126 @@ public void testDomainXMLParser() {
         assertEquals(WatchDogDef.WatchDogModel.I6300ESB, watchDogs.get(0).getModel());
         assertEquals(WatchDogDef.WatchDogAction.RESET, watchDogs.get(0).getAction());
     }
+
+    @Test
+    public void testDomainXMLParser2() {
```

Comment:
how about a more explicit name?
```suggestion
    public void testDomainXMLParserWithoutModelName() {
```
not claiming this name is good or the best by any means.